docker/images/Dockerfile.quickstart

```diff
@@ -0,0 +1,12 @@
+# This file is intended for development only.
+# It will allow users to quickly get this app running so that they can start making updates without worrying about
+# whether or not dependencies are properly installed.
+FROM node:14-alpine
+
+RUN apk update && apk --no-cache add build-base autoconf automake libtool pkgconfig nasm
```

Comment:
[change] Let's also add `util-linux` package.

[suggest] I personally like this format for package installation and alphabetize the packages so it's easier to know what I've installed:

```
RUN apk update && apk --no-cache add \
  autoconf \
  automake \
  build-base \
  libtool \
  nasm \
  pkgconfig \
  util-linux
```